modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/dev/entry.js

```diff
@@ -13,12 +13,10 @@ import {render} from '@liferay/frontend-js-react-web';
 
 import Diagram from '../src/main/resources/META-INF/resources/js/Diagram';
 
-import '../src/main/resources/META-INF/resources/css/diagram.scss';
-
 render(
 	Diagram,
 	{
-		imageURL: './assets/lfr_diagram_engine.png',
+		imageURL: './assets/308056.svg',
```

Comment:
It's hardcoded only because this is only for development purposes, but it's a real use case file.
We know it's a bit slow but will probably be future changes about it